core/spec.md

```diff
@@ -3808,22 +3848,33 @@ and the following Resource level attributes:
   - If present, it MUST be a case sensitive `true` or `false`.
   - If present in a request, a value of `null` has the same meaning as
     deleting the attribute, implicitly setting it to `false`.
-  - Since this attribute and `defaultversionid` are closely related, the
-    processing of them in a request message MUST adhere to the following:
-    - The `defaultversionsticky` attribute is applied first. As a reminder,
-      in the `PATCH` case, if this attribute is missing in the request then
-      this attribute remains unchanged in the Resource.
-    - If the resulting value of this attribute is `false` then the sticky
-      aspect MUST be turned off, and any `defaultversionid` in the request
-      MUST be ignored. The newest Version MUST be the default Version.
-    - If the resulting value of this attribute is `true` then the sticky
-      aspect MUST be turned on, and any `defaultversionid` attribute from
-      the request is applied - where a value of `null` means "newest".
-      If the request was a `PUT` or `POST` and did not have a
-      `defaultversionid` then the implicit value of `null` MUST be used,
-      resulting in "newest". If the request was a `PATCH` and did not have a
-      `defaultversionid` then the current default Version MUST be used.
-      A reference to a Version that does not exist MUST generate an error.
+  - Since this attribute and `defaultversionid` can impact each other, their
```

Comment:
Just a meta comment really... if I were implementing this, I'd have two simultaneous thoughts:

- "I'm really grateful this specified in such a detailed way. I can write tests based on this."
- "Wow, it's really complicated. Does it need to be this complicated?"